app/services/permission_service.rb

```diff
@@ -0,0 +1,52 @@
+require 'pry'
+class PermissionService
+  attr_reader :user
+
+  def initialize(user)
+    @user= user || User.create
+  end
+
+  def allow?(controller)
```

Comment:
@NicoleKMarino we can refactor this method pulling out the conditionals into separate methods (remember what we did in class with Jorge?)